### DISCUSSION TITLE
"\xC3" from ASCII-8BIT to UTF-8 Evaluation Error

### DISCUSSION THREAD

**Discussion by oneindelijk** [choria-io/general#32] [Q&A]
Hi, 

I get this error when a result from either the webhook module or 'curl' with the shell module whenever a special character is returned.
I tested this with a set of records that contained fore example the name 'Joël', after removing the name from the record, it succeeded, otherwise the choriatask would fail with error below:
`Error: Evaluation Error: Error while evaluating a Function Call, "\xC3" from ASCII-8BIT to UTF-8 (file: /etc/puppetlabs/code/to_server.pp, line: 21, column: 19) on node

**Comment by ripienaar** [comment]
@smortex maybe something you have co e across before?

**Comment by smortex** [comment]
Just installed the agent and reproduced the failure.

I think we can fix it that way:
https://github.com/choria-plugins/shell-agent/pull/16

@oneindelijk can you test this?

**Reply by oneindelijk** [reply]
I just discovered we haven't updated choria for three years. I will do this the coming week and perform the tests.
Thanks for your quick actions and reactions, that instills a lot of confidence in our team  about using choria.

**Reply by ripienaar** [reply]
Thank you @smortex and sorry also for me being slow to respond, forgot my laptop on the wrong side of Force 10 winds and no ferries hehe
